Validation/HGCalValidation/plugins/HGCalDigiValidation.cc

```diff
@@ -215,8 +222,11 @@ void HGCalDigiValidation::analyze(const edm::Event& iEvent,
 	const HGCSample&  hgcSample = it.sample(SampleIndx_);
 	uint16_t   gain      = hgcSample.toa();
 	uint16_t   adc       = hgcSample.data();
-	double     charge    = adc*gain;
-	digiValidation(detId, geom0, layer, adc, charge);
+	//double     charge    = adc*gain;
```

Comment:
what motivates this change?

Reply:
To my understanding, I don't think energy weighted TOA is used in the simulation/reconstruction. So I simply change it to TOA. Let me know if I misunderstood.

Reply:
@amartelli @Saptaparna can you take a look and confirm if this makes sense?

Reply:
Hi @kpedro88 @jingyucms, 
I agree that adc*toa is not interesting and not used. 
Instead having a plot dedicated to the ToA is  very useful. 

thanks for this